app-web/plugins/gatsby-source-github-all/utils/constants.js

```diff
@@ -27,14 +27,22 @@ const SOURCE_TYPES = {
   GITHUB: 'github',
 };
 
+const RESOURCE_TYPES = {
+  COMPONENTS: 'Components',
+  DOCUMENTATION: 'Documentation',
+  SELF_SERVICE_TOOLS: 'Self-Service Tools',
+  RESPOSITORIES: 'Repositories',
+  PEOPLE: 'People',
+  PROJECTS: 'Projects',
+};
 // Resource Types for Devhub
-const RESOURCE_TYPES = [
-  'Documentation',
-  'Repositories',
-  'People',
-  'Projects',
-  'Self-Service Tools',
-  'Components',
+const RESOURCE_TYPES_LIST = [
```

Comment:
You can create this as needed with `Object.values(RESOURCE_TYPES)` it has the small advantage of not having to maintain two lists.

Reply:
great point. Changed.